src/expand.rs

```diff
@@ -127,7 +140,7 @@ fn transform_sig(
                 where_token: Default::default(),
                 predicates: Punctuated::new(),
             });
-        for param in &sig.generics.params {
+        for param in sig.generics.params.iter().chain(&context.generics().params) {
```

Comment:
Hmm... [this seems too strict](https://github.com/dtolnay/async-trait/issues/8#issuecomment-514812245).